site/src/template.md

```diff
@@ -0,0 +1,93 @@
+---
+title: Title
+description: Description
+template: standard
+---
+
+# Page title (H1)
+
+The first thing is the general description.
+
+Then the list of some general classes can be listed, if there are not many of them, or css style specific details
+
+_Example_
+
+The following spacing sizes are supported:
```

Comment:
<!--StartFragment-->The following spacing sizes are supported:- `s`- `m`- `l`- `none`<!--EndFragment-->

Seems like the above could be removed. Isn't it fine to reference this in the table?

Reply:
I think it really depends on a specific case, if you need to list just 3 possible values - does it worth it to make the whole table and think about the proper titles for row/columns in these kinda cases?

Reply:
Yes, I think we should try to present the information in a mostly uniform way, and I don't think a list is the right one.